### ISSUE TITLE
odo login seems to erase .kube/config

### ISSUE THREAD

**Issue by jeffmaury** [redhat-developer/odo#992]
[kind/bug]

## What versions of software are you using?

Operating System : Win10 Pro

Output of `odo version` : 
```
odo v0.0.15 (2f7ed497)                             
                                                   
Server: https://api.rh-us-east-1.openshift.com:443 
OpenShift: v3.9.31                                 
Kubernetes: v1.9.1+a0ce1bc657   
```                   

## How did you run odo exactly?

`odo login api.rh-us-east-1.openshift.com --token=`

## Expected behavior
Previous contexts in `.kube/config` should be preserved

## Actual behavior
odo seems to erase the `.kube/config` file and a single cluster/context is then present

## Steps to reproduce the behavior
`oc login cluster1` -> info about cluster 1 are now in `.kube/config`
`odo login cluster2` -> info about cluster2 are now lost

**Comment by kadel** [comment]
verified. This is not good. @surajnarwade

**Comment by surajnarwade** [comment]
@kadel fixed the bug in #997
